build-cwl-docker.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 set -e
+docker build --file=cwltool.Dockerfile --tag=commonworkflowlanguage/cwltool-module --target module .
```

Comment:
I think this is supposed to be "--target builder" or `FROM python:3.6-alpine as builder` should be `FROM python:3.6-alpine as module` ?